src/scores/continuous/murphy_impl.py

```diff
@@ -148,7 +148,7 @@ def _expectile_elementary_score(fcst: FlexibleArrayType, obs: FlexibleArrayType,
 
 def _check_murphy_inputs(alpha=None, functional=None, huber_a=None, left_limit_delta=None):
     """Raise ValueError if the arguments have unexpected values."""
-    if (alpha is not None) and not (0 < alpha < 1):
+    if (alpha is not None) and not (0 < alpha < 1):  # pylint: disable=C0325
```

Comment:
The parenthesis make it clearer to me, so I'm happy to ignore this pylint message.